run-tests-jenkins

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
-coverage run manage.py test -v 3 --failfast --with-xunit $1 --with-timer
+coverage run manage.py test -v 3 --with-xunit $1 --with-timer
```

Comment:
should we keep ```failfast``` parameter to make Jenkins to exit upon failures to reduce test run time?

Reply:
This should go through as Scott coded it, if it's ok Hong... Scott and I discussed this and Phuong has an action item to parameterize this in the build, making failfast the default and giving the developer the option of controlling after logging into Jenkins and specifying a param to not fail fast, such as in Scott's case when many tests were failing.

Reply:
![image](https://user-images.githubusercontent.com/45397838/52972600-646e5800-3389-11e9-830b-82da3a892b8c.png)

Reply:
@cuahsimarko OK. Understood. I'll defer this to you, @phuongdm  and @sblack-usu then.